src/main/flight/imu.h

```diff
@@ -87,3 +87,4 @@ bool imuQuaternionHeadfreeOffsetSet(void);
 void imuQuaternionHeadfreeTransformVectorEarthToBody(t_fp_vector_def * v);
 bool shouldInitializeGPSHeading(void);
 bool isUpright(void);
+const float (*getCraftRotationMaxtrix(void))[3];
```

Comment:
It may be better to hide rotation matrix as implementation detail. Simply export body-to-earth and earth-to-body functions ...